assets/training/finetune_acft_hf_nlp/src/register_model/register_model.py

```diff
@@ -191,9 +191,14 @@ def get_properties(finetune_args_path: str) -> Dict[str, str]:
         if "baseModelId" == property_key:
             properties[property_key] = "/".join(properties[property_key].split('/')[:-2])
 
+    if "baseModelId" in properties:
```

Comment:
Shall we merge it to the _if condition_ in L191 where we check for _baseModelId_?

Reply:
done